src/Tasks/Microsoft.Common.CurrentVersion.targets

```
@@ -4569,6 +4569,10 @@ Copyright (C) Microsoft Corporation. All rights reserved.
       <Output TaskParameter="TargetOutputs" ItemName="_ThisProjectItemsToCopyToOutputDirectory" />
     </CallTarget>
 
+    <ItemGroup Condition="'$(CopyConflictingTransitiveContent)' == 'true'">
```

You did exactly what I said, but I was wrong. Sorry.

The normal behavior is that we copy conflicting transitive content. So we want to explictly stop doing that.

```suggestion
    <ItemGroup Condition="'$(CopyConflictingTransitiveContent)' == 'false'">
```

right?

---

Yes; I agree.

"Ignore" and "Copy" sound like opposites, but it was ignore the _conflicts_ or copy the _files_, which actually does the same thing. I'm wondering if we should ask another opinion on how to make this less confusing.

---

Right, totally missed that, sorry!